mobile_verifier/src/reward_shares.rs

```diff
@@ -776,6 +754,9 @@ fn eligible_for_coverage_map(
     radio_type: RadioType,
     trust_scores: &[LocationTrust],
 ) -> bool {
+    if radio_type == RadioType::IndoorCbrs || radio_type == RadioType::OutdoorCbrs {
```

Comment:
are we maintaining these variants b/c of potentially processing old data or could they just be deleted? it seems like we're no longer ingesting any data that might generate these types of records unless we're maintaining some degree of backward compatibility for existing records

Reply:
Cbrs is removed from RadioType and HbType